doc/index.rst

```diff
@@ -1,12 +1,78 @@
 .. _framework:
 
 =====================
-Core functionalities
+ROS2 Control API
```

Comment:
Please add a separate file for this and keep the existing file as it is. 

The File with API should be referenced then from the indes file. Is this possible?

Reply:
Yes, I can work on this